griptape/tools/file_manager/tool.py

```diff
@@ -45,39 +45,66 @@ def load_files_from_disk(self, params: dict) -> ListArtifact | ErrorArtifact:
         return list_artifact
 
     @activity(config={
-        "description": "Can be used to save an artifact namespace to disk",
+        "description": "Can be used to save data to disk",
         "schema": Schema({
-            "memory_name": str,
-            "artifact_namespace": str,
             Literal(
                 "path",
                 description="Destination path on disk in the POSIX format. For example, ['foo/bar/file.txt']"
```

Comment:
While we're in here making changes...

The description should probably be `Destination path on disk in the POSIX format. For example, 'foo/bar/file.txt'` since it is only a single output path.